src/core/mavlink_commands.h

```diff
@@ -12,10 +12,97 @@ namespace mavsdk {
 
 class SystemImpl;
 
-class MAVLinkCommands {
+class MavlinkCommandReceiver {
 public:
-    explicit MAVLinkCommands(SystemImpl& parent);
-    ~MAVLinkCommands();
+    explicit MavlinkCommandReceiver(SystemImpl& parent);
+    ~MavlinkCommandReceiver();
+
+    struct Command {
+        uint8_t target_system_id{0};
+        uint8_t target_component_id{0};
+        uint16_t command{0};
+
+        // Int
+        MAV_FRAME frame = MAV_FRAME_GLOBAL_RELATIVE_ALT;
```

Comment:
```suggestion
        MAV_FRAME frame {MAV_FRAME_GLOBAL_RELATIVE_ALT};
```

as well as the others below.